dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/CommandLine.java

```diff
@@ -93,7 +93,6 @@ static ArgumentParser getArgumentParser() {
                         "(default: Dockerfile,docker-compose)");
         parser.addArgument("-r", "--" + RATE_LIMIT_PR_CREATION)
                 .type(String.class)
-                .setDefault("")
```

Comment:
Setting this default was causing the flow to throw java.util.UnknownFormatConversionException and proceed when rate limiting was not used https://github.com/salesforce/dockerfile-image-update/blob/main/dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimit.java#L61
Removing this will make the logging more cleaner and give apt info message when rate limiting not used